### DISCUSSION TITLE
Firewall requirements?

### DISCUSSION THREAD

**Discussion by vcmcstech** [trimble-oss/dba-dash#88] [Q&A] [answered]
What firewall rules are required for this?

**Comment by DavidWiseman** [comment, marked as answer]
Most of the data collected from DBA Dash is done via a SQL connection - so port 1433 by default.  There is also some data collected via WMI.  For example, drive space is collected via WMI - to get both drive capacity and free space for all the drives on the server.  It's possible to disable WMI if you want to - you can do this when adding connections or click the "No WMI" option in the "Existing Connections" grid.  If WMI is disabled DBA Dash will collect all data via a SQL connection.  You will still get drive space data - but it might not be for ALL drives on the server.

If you have problems with the WMI collections you should see these in the application Logs folder and also the "DBA Dash Error Log" in the GUI.  If you have errors you should either fix them or disable WMI.

I found some firewall documentation for WMI here:

https://docs.microsoft.com/en-gb/windows/win32/wmisdk/connecting-to-wmi-remotely-starting-with-vista

You also need to have appropriate permissions to run WMI queries.

Hope this helps,

David
